proto/cosmos/base/tendermint/v1beta1/query.proto

```diff
@@ -85,7 +85,7 @@ message GetBlockByHeightRequest {
 // GetBlockByHeightResponse is the response type for the Query/GetBlockByHeight RPC method.
 message GetBlockByHeightResponse {
   .tendermint.types.BlockID block_id = 1;
-  .tendermint.types.Block   block    = 2;
+  Block                     block    = 2;
```

Comment:
We unfortunately can't do that without bumping to v1beta2.

Can we instead add a new field in the Response message?